### DISCUSSION TITLE
A cPath2 instance with Biofactoid-derived BioPAX

### DISCUSSION THREAD

**Discussion by jvwong** [PathwayCommons/factoid#915] [General]
- Outstanding issues
  - [x] hosting 
  - [x] domain

**Comment by jvwong** [comment]
There plan moving forward is to create a separate cPath2 instance on the same VM as the existing PC2 because it has more than sufficient unused resources. Furthermore, we will use the idle `beta.pathwaycommons.org` domain. The data to include will proceed through at least 3 stages:

1. Biofactoid data only
2. Add current PC2 data (v12)
3. New/updated PC data

Does this sound right @metincansiper ?

**Comment by metincansiper** [comment]
@jvwong Do we need it working with current pc2 data (step 2) and new/updated pc2 data (step 3) separately? If step 2 is just a preparation for step 3 then may I just directly do step3 by skipping step2?

**Comment by metincansiper** [comment]
I prepared some of the data however I could/did not do some of them yet. Because they require membership (both paid and free) or the resource is not available. Are there memberships that are already taken for PC? Should I enroll to the free ones using my personal info? For the unavailable/not found resources I may try to reach out the owners. Could we consider using the archived PC2 v12 intermediate data if they are totally unavailable or shall we just exclude them? Below are the list of the data I mentioned:

**Requires membership (some free, some paid)**
- humancyc - https:\/\/humancyc.org\/download.shtml (paid)
- hprd  - http:\/\/www.hprd.org\/download
- dip - https:\/\/dip.doe-mbi.ucla.edu\/dip\/Download.cgi (I think the ones that we need requires membership)
- drugbank - https:\/\/www.drugbank.ca\/releases\/5-1-4\/downloads\/all-full-database


**Data not available or I could not find**
- mirtarbase - http:\/\/mirtarbase.mbc.nctu.edu.tw\/cache\/download\/7.0\/hsa_MTI.xlsx (website is not reachable)
- ctd - http:\/\/ctdbase.org\/downloads\/ (I could not file an owl file like the one used in archived intermediate pc2 data I may need to search more for that)
- kegg - http:\/\/www.cogsys.cs.uni-tuebingen.de\/mitarb\/draeger\/BioPAX.zip
- inoh  - http:\/\/inoh.hgc.jp\/download.html#PathwayData (server is not accessible but some archive is available at - https://dbarchive.biosciencedbc.jp/jp/inoh/download.html. I suspect that the archive files I found may already be the same version of the ones used in pc2 v12)
- netpath  - http:\/\/www.netpath.org\/download\/zipped\/BioPAX.zip (batch download link is available but corresponding page is not available)
